src/nimgen/c2nim.nim

```diff
@@ -53,7 +57,7 @@ proc c2nim*(fl, outfile: string, c2nimConfig: c2nimConfigObj) =
   for prag in c2nimConfig.pragma:
     outpragma &= "{." & prag & ".}\n"
 
-  let fname = file.splitFile().name.multiReplace([(".", "_"), ("-", "_")])
+  let fname = file.splitFile.name.normalize.capitalizeAscii
 
```

Comment:
Normalize doesn't get rid of -. We want to make a filename into something that can be used as an identifier.

Reply:
Maybe switch this around? First replace . and - and then normalize? Otherwise you still have _? Not exactly sure what's the purpose of this change.

Reply:
Fixed.

Reply:
Still looks the same. I'm suggesting:

```nim
  let fname = file.splitFile.name.multiReplace([(".", "_"), ("-", "_")]).normalize.capitalizeAscii
```

Reply:
Done.